### DISCUSSION TITLE
Open Usage Graphs in separated borderless window

### DISCUSSION THREAD

**Discussion by HardRock88** [cockpit-project/cockpit#16037] [Ideas]
Hello, I am only starting to digging into Cockpit.
I want a simple link for opening "Usage Graphs" in separated window, preferable border less.
So I can put it on second monitor.
results should be something like this:
![image](https://user-images.githubusercontent.com/5637425/124335244-645e0400-dba2-11eb-9a69-643920d8e061.png)
But window title needed to be named as monitored PC name and breadcrumbs should be removed.
And It will be very cool to add quick links for every server to top panel, it is empty any way.

First I wanted to create a plugin for this, but so far I can not really understand where to start to do such simple task.
For now I am only find out that this page is building here
cockpit/pkg/metrics/
But it seems that this code can't be reused in custom module, should I edit sources for this ?
How can I add simple link for opening graphs in separated window named by server name and without breadcrumbs ?

**Comment by garrett** [comment]
If you just want to remove or tweak parts of the UI, you could consider a userscript browser add-on, such as Stylus:
https://addons.mozilla.org/en-US/firefox/addon/styl-us/
https://chrome.google.com/webstore/detail/stylus/clngdbkpkpeebahjckkjfobafhncgmne

...or Stylebot (which is similar in concept, but more visual): https://stylebot.dev/

I'm not sure if they'd work with Cockpit's CSP, however.

If you're interested in dashboards, Cockpit can set up and export a backend for Grafana now: https://cockpit-project.org/blog/pcp-grafana.html

I guess you're asking for the server name above the dashboard? Integrating it makes sense.

**Reply by martinpitt** [reply]
You can show just the metrics frame instead of the whole cockpit UI. https://cockpit-project.org/guide/latest/embedding.html#embedding-components describes how: http://localhost:9090/cockpit/@localhost/metrics/index.html . Replace "localhost" with the server you want to monitor.

**Reply by martinpitt** [reply]
Alternative: `/usr/libexec/cockpit-desktop metrics`, or `/usr/libexec/cockpit-desktop metrics ssh_target`

**Reply by HardRock88** [reply]
cool, I will look into it, thx )
